.eslintignore

```diff
@@ -1 +1,2 @@
-dist
\ No newline at end of file
+dist
+node_modules
```

Comment:
> https://eslint.org/docs/user-guide/configuring/ignoring-code

Repetitive, this is already ignored by default.

Reply:
Awesome, I didn't know that, so thanks!